src/server/report/test.js

```diff
@@ -185,6 +184,71 @@ class Test extends LitElement {
 		.warning {
 			color: #e87511;
 		}
+		.result {
+			padding: 20px;
+		}
+		.result-split {
+			flex-direction: row;
+			flex-wrap: nowrap;
+			display: flex;
+		}
+		.result-split > .result-part {
+			flex: 0 1 auto;
+		}
+		.result-split-divider {
+			border-right: 4px dashed #007bff;
+			flex: 0 0 auto;
+		}
+		.result-part {
+			display: inline-block;
+		}
+		.result-diff-container img {
+			max-width: 100%;
+		}
+		.result-diff-container {
+			background: repeating-conic-gradient(#cdd5dc 0% 25%, #ffffff 0% 50%) 50% / 20px 20px;
+			background-position: 0 0;
+			border: 2px dashed #90989d;
+			display: inline-block;
+			line-height: 0;
+			position: relative;
+		}
+		.result-split > .result-part:first-of-type > .result-diff-container {
+			border-right: none;
+		}
+		.result-split > .result-part:last-of-type > .result-diff-container {
+			border-left: none;
+		}
+		.result-overlay {
+			background: hsla(0,0%,100%,.8);
+			position: absolute;
+			top: 0;
+			left: 0;
+		}
+		.result-part-info {
+			align-items: center;
+			display: flex;
+			gap: 5px;
+		}
+		.result-part-info-spacer,
+		.result-part-info-size {
+			flex: 1 0 0%;
+			text-align: right;
```

Comment:
I think I like this slightly better `text-align: left;` (and `gap` reduced) but I see the benefit of having both at the ends